src/hooks/useTokenBalanceToUsd.js

```diff
@@ -86,3 +93,42 @@ export default function useANJBalanceToUsd(amount) {
 
   return convertedAmount
 }
+
+/**
+ * Convert a token into a USD price
+ *
+ * @param {String} symbol The symbol of the token to convert from.
+ * @param {Number} decimals The amount of decimals for the token.
+ * @param {BigNumber} balance The balance to convert into USD.
+ * @returns { Number } The balance value in USD
+ */
+export default function useTokenBalanceToUsd(symbol, decimals, balance) {
```

Comment:
Hmm do we use this? Do we show any other tokens in the dashboard?

Reply:
yes ANT and DAI